support-frontend/assets/helpers/user/userReducer.js

```diff
@@ -21,6 +21,7 @@ export type User = {
   isSignedIn: boolean,
   isRecurringContributor: boolean,
   emailValidated: boolean,
+  isReturningContributor: boolean,
```

Comment:
is the date of the contribution of any use for this upcoming test?

Reply:
I don't think we should reveal the date or amount unless they are signed in